packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -5,7 +5,6 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
```

Comment:
FYI the code in this was a big mess, with many parts that could be refactored for reusing, leading to simpler/smaller file.

1. Since VSCode extension does not start the server anymore and our Preview command handles all 3 platform including Desktop, I removed all of the tests for those (since the product code for that is removed and our unit tests for Preview command in other repos cover those cases too)

2. Refactored a lot of the test code into reusable chunks to simplify the test code.